src/store/slices/blogsSlice.js

```diff
@@ -0,0 +1,175 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const blogsSlice = createSlice({
```

Comment:
Can you please explain why are you populating this slice.

Reply:
And because of fact that for functional components is ideal to have zero number of state passed through props, we use redux toolkit to menage blogs state by making slices for state.